plugins/catalog-import/report.api.md

```diff
@@ -140,11 +142,13 @@ export { catalogImportPlugin };
 export { catalogImportPlugin as plugin };
 
 // Warning: (ae-forgotten-export) The symbol "StepperProvider" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "catalogImportTranslationRef" needs to be exported by the entry point index.d.ts
```

Comment:
This should be exported

Reply:
I just added it into `alpha.ts` entry like other plugins.